assets/js/profile.js

```diff
@@ -105,8 +112,31 @@ showHistory();
 // History Reset Button
 const historyResetBtn = document.getElementById("historyResetBtn");
 historyResetBtn.addEventListener("click", () => {
-    localStorage.removeItem("typerName");
+    //localStorage.removeItem("typerName");
     localStorage.removeItem("typerHistory");
     showHistory();
-    window.location.reload();
+   // window.location.reload();
+   //to show msg after delete data
+   checkIfUserData(" to practice again")
+})
+
+//after click on delete username button 
+const deleteUserName=document.querySelector(".reset-userName")
+  deleteUserName.addEventListener("click",(e)=>{
+    localStorage.removeItem("typerName");
+    e.target.style.display="none"
+    checkIfUserData(" to set up your name and practice again")
```

Comment:
if the user visits the website for the first time. The text appears will ' to set up your name and practice again '. the word again didn't make any sense as the user visits for the first time. so remove it.

Reply:
done